### PR TITLE
changed to nsls2forge channel and added test to recipe

### PR DESCRIPTION
**Pull request by buijennifer**

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ github:
   user_or_org: nsls-ii-forge
 channels:
   sources:
-    - lightsource2-tag
+    - nsls2forge
     - defaults
```
Comment (CJ-Wright) [inline conda-forge.yml:6]: Should this include conda-forge as well?

Comment (mrakitin) [reply conda-forge.yml:6]: Not now, sorry. We are trying to produce a set of packages using `defaults` and `nsls2forge` for now.

**Review by mrakitin — COMMENTED** [review]
Should we also bump a version number to allow uploading of the new builds?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,22 +9,28 @@ source:
   git_rev: v{{ version }}
 
 build:
-  number: 8
+  number: 9
```
Comment (mrakitin) [inline recipe/meta.yaml]:
```suggestion
  number: 9
```

**Review by mrakitin — COMMENTED** [review]
Looks good overall, suggested to remove a single blank space.

**Comment by mrakitin** [comment]
All the builds passed. Merging now.

**Review by tacaswell — APPROVED** [review]

**Pull request merged**